### ISSUE TITLE
Unable to create multiple consumers for different topics  in the same process?

### ISSUE THREAD

**Issue by mango009** [confluentinc/confluent-kafka-go#1313]
I create multiple consumers running in different Go routines. These consumers consume messages on different topics.

But I notice that when one consumer is blocked processing messages, the other consumer would get timeout error on the `ReadMessage` function.

Is it not a practice to create multiple consumers in the same process?

**Comment by OneCricketeer** [comment]
Does each routine run one consumer group? 

One consumer can already subscribe to a regex pattern of topic names. 

Sub-processing is an alternative solution than coroutines

**Comment by mango009** [comment]
Yes. Each consumer group run in it's own go routine. I know alternative is to subscribe multiple topic in one go routine. But the API doesn't preclude subscribing one topic in separate go routine. And if this is a bad practice it would helpful to state this in the document (and maybe give a reason).

**Comment by OneCricketeer** [comment]
Can you please share an example? Also, do they all share a config? Specifically all a single `group.id`
